ui/scripts/renderTypePageTest.js

```diff
@@ -0,0 +1,214 @@
+const faker = require('faker')
+faker.seed(1)
+const mapObject = require('./render-utils').mapObject
+
+function renderTypePageTest (typeName, fields) {
+  const name = typeName
+  const namePlural = name + 's'
+  const capsName = typeName.toUpperCase()
+  const capsNamePlural = capsName + 'S'
+  const lowerName = typeName.toLowerCase()
+  const lowerNamePlural = lowerName + 's'
```

Comment:
I'm using a similar approach for naming in the dna gen.  What do you think about just extracting these into a name/string alteration function that can be referenced from a shared utils folder as they are referenced in multiple places? :+1: